scripts/docker/install_os_dependencies.sh

```diff
@@ -125,13 +133,35 @@ function install_debian_runtime_dependencies() {
     rm -rf /var/lib/apt/lists/* /var/log/*
 }
 
+function install_python() {
+    git clone --branch "${AIRFLOW_PYTHON_VERSION}" --depth 1 https://github.com/python/cpython.git
+    cd cpython
+    ./configure --enable-optimizations
```

Comment:
```suggestion
    ./configure --enable-optimizations --prefix=/usr/python/
 ```

For example. Then we could add PATH, LD_LIBRARY_PATH and maybe few other places to make sure  "/usr/python/bin", "/usr/python/lib" etc are first on the PATHs.

Reply:
Yeah, that makes a lot of sense. I'll look into that later today.

Reply:
Thanks for the idea!